src/app/features/tournament-list/tournament-list.component.ts

```diff
@@ -7,15 +7,17 @@ import { TournamentService } from 'src/app/shared/services/tournament.service';
   styleUrls: ['./tournament-list.component.scss']
 })
 export class TournamentListComponent implements OnInit {
-  imageSource = '../../../assets/images/head-bg.jpg';
   tournamentList = [];
   pageActual = 1;
-  
+  @Input() flag;
+
   constructor(private tournamentService: TournamentService) { }
 
   ngOnInit() {
     this.tournamentService.get().subscribe(tournamentList => {
-      this.tournamentList = tournamentList;
+      this.flag 
```

Comment:
`this.tournamentList = this.flag ? tournamentList.slice(0, 3) : tournamentList;`